purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/GalaxyBillingWrapper.kt

```diff
@@ -138,19 +138,30 @@ internal class GalaxyBillingWrapper(
     ) {
         val galaxyPurchaseInfo = purchasingData as? GalaxyPurchasingData.Product
         if (galaxyPurchaseInfo == null) {
+            val errorMessage = PurchaseStrings.INVALID_PURCHASE_TYPE.format(
+                "Galaxy",
+                "GalaxyPurchasingData",
+            )
             val error = PurchasesError(
                 PurchasesErrorCode.UnknownError,
-                PurchaseStrings.INVALID_PURCHASE_TYPE.format(
-                    "Galaxy",
-                    "GalaxyPurchasingData",
-                ),
+                errorMessage,
             )
-            errorLog(error)
+            log(LogIntent.GALAXY_ERROR) { errorMessage }
```

Comment:
I realized that `errorLog` can use the `GOOGLE_ERROR` log intent, so this sets it to use the correct log intent.